LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/LoRaDeviceClientConnectionManager.cs

```diff
@@ -93,7 +93,7 @@ public void Register(LoRaDevice loRaDevice, ILoRaDeviceClient loraDeviceClient)
                         try
                         {
                             using var scope = logger.BeginDeviceScope(client.DevEui);
-                            await client.DisconnectAsync();
+                            await client.DisconnectAsync(CancellationToken.None);
```

Comment:
@p-schuler should we force the disconnection here?

Reply:
Yes - I think we should - the point here is that after a certain amount of time we want to disconnect. Since it is a sliding expiration, we will only disconnect if we haven't had any messages from that device for that amount of time. Of course the memory cache makes it non-deterministic, but once we hit that point, even if we own it, we should close it.

Reply:
I just realized that this is the synchronized client, so it should be fine as it is.
Agree?

Reply:
yeap - the disconnectAsync on the LoRaDeviceClient(s) is always executed. Only the implementation in LoRaDevice is taking the ownership into consideration. all good.